presto-common/pom.xml

```diff
@@ -102,5 +102,17 @@
             <artifactId>jackson-core</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.facebook.drift</groupId>
+            <artifactId>drift-codec</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.facebook.drift</groupId>
+            <artifactId>drift-protocol</artifactId>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
-</project>
+</project>
```

Comment:
stray new line?

Reply:
For some reason adding this new line causes it to fail Maven tests for check-other-modules… any idea what might cause this?

Reply:
I think the error is not related. Can you try rebasing with master once. We had several flaky tests in the past. Maybe this has been fixed in some of the recent commits.